.gitignore

```diff
@@ -9,6 +9,7 @@ npm-debug.log
 .eslintcache
 package-lock.json
 /tools/set-import/sets
+database/sqlite.db
```

Comment:
I think we agreed on `databases`?

Also should we just ignore a `databases/` and force important files into Git?

Reply:
oh, I might have messed that up. I think it should be plural though because logs is plural as well. And yes, ignore databases/*.db* or ignore databases and !databases/migrations or whatever

Reply:
> I think we agreed on databases?

There was never any discussion on it as far as I'm aware. I don't particularly care about it.

> Also should we just ignore a databases/ and force important files into Git?

I think `database(s)/*.db` is a better suggestion than just ignoring everything.

Reply:
Sure. But there's nothing wrong with ignoring everything; it's what we do in `config` and `logs` after all.